Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -207,18 +232,47 @@ class ApplicationManifestParserTest : public testing::Test {
         EXPECT_EQ(expectedValue, value);
     }
 
+    void testCategories(const Vector<String>& expectedValue)
+    {
+        StringBuilder builder;
+        builder.append('[');
+        auto expectedValueSize = expectedValue.size();
+        for (auto& value : expectedValue) {
+            builder.append('\"');
+            builder.append(value);
+            builder.append('\"');
+            auto builderLength = builder.length();
+            if (builderLength > 1 && builderLength < expectedValueSize)
```

Comment:
This isn't what I had suggested. Why the change?

Reply:
My proposal was to move it to *before* we append the quoted value and with only one `builderLength > 1` being checked. What you need us with doesn't make sense. `builderLength > 1` is always true in your case.

Reply:
Apologies! I misunderstood your suggestion and didn't realize it's meant to be moved before we append the quote. Just updated the PR to move it before we append the quote and drop the unnecessary `builderLength < expectedValueSize` check.